src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -20,11 +20,10 @@
         "composer-runtime-api": ">=2.1",
         "ext-xml": "*",
         "symfony/cache": "^5.4|^6.0",
-        "symfony/config": "^5.4|^6.0",
-        "symfony/dependency-injection": "^5.4.5|^6.0.5",
+        "symfony/config": "^6.1",
+        "symfony/dependency-injection": "^6.1",
         "symfony/deprecation-contracts": "^2.1|^3",
         "symfony/event-dispatcher": "^5.4|^6.0",
-        "symfony/error-handler": "^5.4|^6.0",
```

Comment:
why removing the direct requirement ? FrameworkBundle is using the error-handler component directly.

Reply:
added back